src/mono/mono/component/debugger-agent.c

```diff
@@ -6517,6 +6543,28 @@ get_types_for_source_file (gpointer key, gpointer value, gpointer user_data)
 	}
 }
 
+static void 
+send_enc_delta (MonoImage *image, gconstpointer dmeta_bytes, int32_t dmeta_len, gconstpointer dpdb_bytes, int32_t dpdb_len)
+{	
+	//TODO: if it came from debugger we don't need to pass the parameters back, they are already on debugger client side.
```

Comment:
This is related to issue https://github.com/dotnet/runtime/issues/55228 - we can assume that if the debugger is attached, all updates are coming through the debugger command, not through the managed icall.

Reply:
I cannot change this right now for WASM, am I right?
Because for wasm the updates are not coming through debugger.

Reply:
That's right. I think for wasm the updates are always through `dotnet watch`, and not through the debugger.